### PR TITLE
GOAL pool registration

### PR DESCRIPTION
**Pull request by dmlebany**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1952,5 +1952,20 @@
...
+  {
+    "ticker":"GOAL",
+    "poolId":"pool109ee6weyn52awvu30gspwmp6xwws98z0dhzcw2dqupc6vv6mkg6",
+    "social":{
+       "facebook":"",
+       "twitter":"",
+       "website":"https://goalstaking.com/",
+       "reddit":"",
+       "instagram":"",
+       "youtube":""
```
Comment (leo42) [inline registry.json:1965]: We need at least one social media

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json [hunk 2]
```diff
@@ -1952,5 +1952,20 @@
     "telegram":"manizzler",
     "operatorName":"manizzler",
     "github":"manizzle"
+  },
+  {
+    "ticker":"GOAL",
+    "poolId":"pool109ee6weyn52awvu30gspwmp6xwws98z0dhzcw2dqupc6vv6mkg6",
+    "social":{
+       "facebook":"",
+       "twitter":"",
```
Comment (dmlebany) [inline registry.json:1961]:
```suggestion
       "twitter":"https://twitter.com/goal_pool",
```

File: registry.json (hunk 1 above)
Comment (dmlebany) [reply registry.json]: @leo42 Added twitter account.

File: registry.json [hunk 3]
```diff
@@ -1952,5 +1952,20 @@
     "telegram":"manizzler",
     "operatorName":"manizzler",
     "github":"manizzle"
+  },
+  {
+    "ticker":"GOAL",
+    "poolId":"pool109ee6weyn52awvu30gspwmp6xwws98z0dhzcw2dqupc6vv6mkg6",
```
Comment (leo42) [inline registry.json:1958]: poolID = 79739d3b249d15d733917a20176c3a339d029c4f6dc58729a0e071a6

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 3 above)
Comment (nikolaypavlov) [inline registry.json:1958]:
```suggestion
    "poolId":"79739d3b249d15d733917a20176c3a339d029c4f6dc58729a0e071a6",
```

Comment (nikolaypavlov) [inline registry.json:1958]:
```suggestion
    "poolId":"79739d3b249d15d733917a20176c3a339d029c4f6dc58729a0e071a6",
```

Comment (dmlebany) [reply registry.json]: Fixed pool id

**Comment by dmlebany** [comment]
There is bunch of conflicts here that I'm not sure how to solve. I'm going to create a new PR with all the recent changes.

**Pull request closed without merging**